modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/database/H2TreeIndexBase.java

```diff
@@ -157,7 +154,7 @@ static int computeInlineSize(
             return cfgInlineSize;
         }
 
-        return Math.min(PageIO.MAX_PAYLOAD_SIZE, size);
+        return size;
```

Comment:
I don't understand this change. What if the computed inline size (not the one set through configuration) will exceed the maximum allowed inline size?